src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
+        <div id="top">
+          <h1 style="text-align: center">John Roberts</h1><br>
+          <h2 style="text-transform: uppercase;text-align: center">business expert</h2>
+          <img src="touxiang.jpg" alt="picture" style="text-align:center">
+        </div>
+
+        <div>
+          <div id="Slider">
+            <!--Slider About me-->
+            <div id="AboutMe_Container">
+              <div id="AboutMe_title">
+                <h2 style="text-transform:uppercase">About Me</h2>
+              </div>
+              <div id=AboutMe_Icon>
+                <ul>
+                  <li style="display:inline">icon1</li>
+                  <li style="display:inline">icon2</li>
+                  <li style="display:inline">icon3</li>
+                </ul>
```

Comment:
Good use of List tags 👍

Reply:
just a suggestion: when you find you are repeating yourself, it's the time to do some abstractions. You should put these same styles into css files.